core/storage/collection/gae_models.py

```diff
@@ -30,12 +30,18 @@
 
 class CollectionSnapshotMetadataModel(base_models.BaseSnapshotMetadataModel):
     """Storage model for the metadata for a collection snapshot."""
-    pass
+    @staticmethod
```

Comment:
Here and elsewhere: add one line above this one to separate it from the class docstring.

Reply:
Done.

Reply:
This is not yet done.

Reply:
@seanlip I see inconsistencies with some models not having this and others having this. I'll make the changes to the files I changed, but I think for the future this should be a linting rule.

Reply:
SG, thanks. I just added it to #8243!